docs/developersguide/testing-practices.rst

```diff
@@ -0,0 +1,72 @@
+Testing practices
+=================
+
+It is a good practice to add test cases to each Invenio module. Tests are
+normally organized in:
+
+1. Unit functional testing: test cases run against the Flask app defined in the
```

Comment:
I agree, depending on which module, tests are a bit of a mess (for example [dojson](https://github.com/inveniosoftware/dojson/blob/master/tests/test_utils.py)
has tests that looks like `unit tests`, but most of the modules usually have more like `functional tests`).
How about we say: `unit and functional` tests for the first bullet and `end-to-end` tests for the second bullet? I'm also perfectly fine with leaving this line as it is, if you think there will be no confusion about it :)